pkg/restapi/vc/operation/operations.go

```diff
@@ -69,17 +77,26 @@ type Client interface {
 }
 
 // New returns CreateCredential instance
-func New(provider storage.Provider, client Client, kms legacykms.KMS, vdri vdriapi.Registry) (*Operation, error) {
+func New(provider storage.Provider, client Client, kms legacykms.KMS, vdri vdriapi.Registry,
+	hostURL string) (*Operation, error) {
 	store, err := provider.OpenStore(credentialStore)
 	if err != nil {
 		return nil, err
 	}
 
+	c := crypto.New(kms, verifiable.NewDIDKeyResolver(vdri))
+
+	vcStatusManager, err := cslstatus.New(provider, hostURL, 50, c)
```

Comment:
List size should be configurable, or at least constant

Reply:
done